plugins/bulk_upload/xml/batch/BulkUploadEngineXml.php

```diff
@@ -718,6 +719,20 @@ protected function sendItemReplaceData($entryId, KalturaBaseEntry $entry ,Kaltur
 			$flavor = KBatchBase::$kClient->flavorAsset->add($updatedEntryId, $flavorAsset);
 			KBatchBase::$kClient->flavorAsset->setContent(KBatchBase::$kClient->getMultiRequestResult()->id, $flavorResource);		// TODO: use flavor instead of getMultiRequestResult
 		}
+
+		$requestResults = KBatchBase::$kClient->doMultiRequest();
+		if(is_array($requestResults))
+			foreach($requestResults as $requestResult)
+			{
+				if(is_array($requestResult) && isset($requestResult['code']))
```

Comment:
I see this same validation is happening few times in the code so why not move it into a function?